CMakeLists.txt

```diff
@@ -36,8 +44,12 @@ install(
     FILES ${PROJECT_BINARY_DIR}/cmcstl2-config.cmake
     DESTINATION lib/cmake/cmcstl2)
 
-add_subdirectory(examples)
+if(STL2_NOT_SUBPROJECT OR STL2_BUILD_EXAMPLES)
```

Comment:
Why test `STL2_NOT_SUBPROJECT` here and on 51 rather than testing only `STL2_BUILD_foo`?